File: ed2/gara2/truffa/att/truffa.c
#include <stdio.h>
#include <assert.h>

#define MAXN 100000

int sfangate(int N, int V[]) {
    // Mettete qui il codice della soluzione
    return 42;
}


int V[MAXN];

int main() {
    FILE *fr, *fw;
    int N, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%d", &N));
    for(i=0; i<N; i++)
        assert(1 == fscanf(fr, "%d", &V[i]));

    fprintf(fw, "%d\n", sfangate(N, V));
    fclose(fr);
    fclose(fw);
    return 0;
}

File: ed2/gara2/truffa/att/truffa.pas
function sfangate(N: longint; var V: array of longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    sfangate := 42;
end;


const
    MAXN = 100000;
var
    N, i  : longint;
    V     : array[0..MAXN-1] of longint;
    fr, fw: text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N);
    for i:=0 to N-1 do
        read(fr, V[i]);

    writeln(fw, sfangate(N, V));
    close(fr);
    close(fw);
end.